backend/app/src/main/kotlin/io/tolgee/api/v2/hateoas/translations/TranslationHistoryModel.kt

```diff
@@ -0,0 +1,33 @@
+package io.tolgee.api.v2.hateoas.translations
+
+import io.swagger.v3.oas.annotations.media.Schema
+import io.tolgee.api.v2.hateoas.user_account.SimpleUserAccountModel
+import io.tolgee.constants.MtServiceType
+import io.tolgee.model.enums.TranslationState
+import org.hibernate.envers.RevisionType
+import org.springframework.hateoas.RepresentationModel
+import org.springframework.hateoas.server.core.Relation
+
+@Suppress("unused")
+@Relation(collectionRelation = "revisions", itemRelation = "revision")
+open class TranslationHistoryModel(
+  @Schema(description = "Translation text")
+  val text: String?,
+
+  @Schema(description = "State of translation")
+  val state: TranslationState?,
+
+  @Schema(description = "Was translated using Translation Memory or Machine translation service?")
+  val auto: Boolean?,
```

Comment:
Isn't this too complicated, I thought that we will store history purely as just text changes.

Reply:
I think that information about state changes or auto translation dismissal would be also something users should know. Also from backend point of view, it's simple to return everything then just particular changes.